apple/testing/default_runner/ios_xctestrun_runner.bzl

```diff
@@ -182,7 +182,8 @@ ios_unit_test(
 ```
 
 If you would like this test runner to generate xcresult bundles for your tests,
-pass `--test_env=CREATE_XCRESULT_BUNDLE=1`
+pass `--test_env=CREATE_XCRESULT_BUNDLE=1`. It is preferable to use the `create_xcresult_bundle`
+on the test runner itself instead of this parameter.
```

Comment:
Line length nit:
```suggestion
pass `--test_env=CREATE_XCRESULT_BUNDLE=1`. It is preferable to use the
`create_xcresult_bundle` on the test runner itself instead of this parameter.
```